### PR TITLE
BathroomFilterCard

### PR DESCRIPTION
**Pull request by grenti**
[Story](https://rentpath.leankit.com/card/583476572)

* Created BathroomFilterCard for react-ui-ag
* Created Inline and Desktop examples for storybook

### CODE REVIEW

File: packages/react-ui-ag/src/Filters/BathroomFilterCard.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+@themed(/^BathroomFilterCard/)
+export default class BathroomFilterCard extends Component {
+  static propTypes = {
+    className: PropTypes.string,
+    theme: PropTypes.object,
+    title: PropTypes.string,
+    description: PropTypes.string,
+    fields: PropTypes.arrayOf(
+      PropTypes.shape({
+        label: PropTypes.string,
```
Comment (tadjohnston) [inline packages/react-ui-ag/src/Filters/BathroomFilterCard.js]: `label` should match the `Field` label type.
```
label: PropTypes.oneOfType([
  PropTypes.node,
  PropTypes.func,
  PropTypes.object,
])
```

File: packages/react-ui-ag/src/Filters/__tests__/BathroomFilterCard-test.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  })
+
+  it('allows data-tid to be overriden', () => {
+    const wrapper = shallow(
+      <BathroomFilterCard
+        data-tid="bathroom-filter-card-test"
+      />
+    )
+    expect(wrapper.find('[data-tid="bathroom-filter-card-test"]')).toHaveLength(1)
+    expect(wrapper.find('[data-tid="bathroom-filter-card"]')).toHaveLength(0)
```
Comment (tadjohnston) [inline packages/react-ui-ag/src/Filters/__tests__/BathroomFilterCard-test.js]: Nice tests!

File: packages/react-ui-ag/src/Filters/__tests__/BathroomFilterCard-test.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  })
+
+  it('renders radiogroups by default', () => {
```
Comment (tadjohnston) [inline packages/react-ui-ag/src/Filters/__tests__/BathroomFilterCard-test.js]: Not sure if we need this. This test is the same as the first.

File: packages/react-ui-ag/src/Filters/BathroomFilterCard.js (hunk 1 above)
Comment (grenti) [reply packages/react-ui-ag/src/Filters/BathroomFilterCard.js]: Good catch - forgot that one! 👍

File: packages/react-ui-ag/src/Filters/__tests__/BathroomFilterCard-test.js (hunk 3 above)
Comment (grenti) [reply packages/react-ui-ag/src/Filters/__tests__/BathroomFilterCard-test.js]: You're right - I was supposed to take that out too. Removing 👍

File: stories/theme/ag/small/Filters/BedroomFilterCard.css [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.InlineCard.BedroomFilterCard-noValue .ApplyButton {
+  display: none;
+}
```
Comment (tadjohnston) [inline stories/theme/ag/small/Filters/BedroomFilterCard.css]: New line needs added here.

File: stories/ag/Filters/BathroomFilterCard.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+const DesktopBathroomFilterCardComponent = ({ theme }) => (
+  <BathroomFilterCard
+    className={theme.SearchFilter}
+    title=""
+    description=""
+    applyButton={{ onClick: value => action('click')(`applied(${value})`) }}
+    cancelButton={{ onClick: () => action('click')('cancelled') }}
+  />
+)
+DesktopBathroomFilterCardComponent.propTypes = {
```
Comment (tadjohnston) [inline stories/ag/Filters/BathroomFilterCard.js]: Should probably put new lines around this.

File: stories/ag/Filters/BathroomFilterCard.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  theme: PropTypes.object,
+}
+
+const ThemedInlineBathroomFilterCard = themed(['InlineCard'])(InlineBathroomFilterCardComponent)
+const InlineBathroomFilterCard = <ThemedInlineBathroomFilterCard />
+
+const DesktopBathroomFilterCardComponent = ({ theme }) => (
+  <BathroomFilterCard
+    className={theme.SearchFilter}
+    title=""
```
Comment (yasabere) [inline stories/ag/Filters/BathroomFilterCard.js]: Is there another way to show no text without leaving props with no value

Comment (grenti) [reply stories/ag/Filters/BathroomFilterCard.js]: Yup. I changed out the defaults now. Good catch 👍

**Pull request merged**